.github/workflows/ci.yml

```diff
@@ -0,0 +1,17 @@
+name: Node.js CI
+
+on: 
+  push:
+    branches: 
+      - '*'
+jobs:
+  build:
+    runs-on: ubuntu-18.04
+    name: Test
+    steps:
+    - uses: actions/checkout@v2
+    - uses: actions/setup-node@v1
+      with:
+        node-version: '10'
+    - run: yarn
+    - run: yarn test
```

Comment:
Lack of newline will cause git to think that this line is changed next time something is appended. It's a minor issue, but it can be easily solved by always adding newline.

It can be "suggested" to editors by adding this to `.editorconfig`:
```diff
 [**/**.yml]
 indent_style = space
 indent_size = 2
+insert_final_newline = true
 
 [**/**.js]
 indent_style = space
 indent_size = 2
+insert_final_newline = true
 
 [**/**.json]
 indent_style = space
-indent_size = 2
+indent_size = 2
+
```
(and `.editorconfig` is lacking this newline as well).